backend/database/migrations/2023_01_23_014012_create_questions_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use App\Models\Question;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('questions', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->longText('content');
+            $table->integer('status')->default(Question::OPEN);
+            $table->boolean('is_public')->default(true);
+            $table->timestamps();
+            $table->softDeletes();
+
+            $table->foreignId('user_id')->constrained()->cascadeOnDelete();
+            // $table->foreignId('team_id')->nullable()->constrained()->nullOnDelete();
```

Comment:
I think no need to use `cascadeOnDelete()` since a user account cannot be deleted?

Also, please remove all non-generated code comments before pushing changes or creating a PR 🙇‍♂️